lms/djangoapps/certificates/views.py

```diff
@@ -224,3 +232,227 @@ def update_example_certificate(request):
+    context['certificate_verify_url'] = _("{prefix}{uuid}{suffix}").format(
+        prefix=context.get('certificate_verify_url_prefix'),
+        uuid=certificate.verify_uuid,
+        suffix=context.get('certificate_verify_url_suffix')
+    )
+    context['logo_alt'] = company_name
+
+    accd_course_org_html = '<span class="detail--xuniversity">{org}</span>'.format(org=course.org)
+    accd_company_name_html = '<span class="detail--company">{company_name}</span>'.format(company_name=company_name)
+    # Translators:
+    context['accomplishment_copy_course_description'] = _('a course of study offered by {org}, through {name}.').format(
+        org=accd_course_org_html,
+        name=accd_company_name_html
+    )
+
+    # Translators:
+    context['accomplishment_more_title'] = _("More Information About {user_name}'s Certificate:").format(
+        user_name=user_fullname
+    )
+
+    # Translators:
+    context['certificate_date_issued_title'] = _("Issued On:")
+
+    # Translators:
+    context['certificate_date_issued'] = _('{month} {day}, {year}').format(
+        month=certificate.modified_date.strftime("%B"),
+        day=certificate.modified_date.day,
+        year=certificate.modified_date.year
+    )
+
+    # Translators:
+    context['certificate_id_number_title'] = _('Certificate ID Number')
+
+    # Translators:
+    context['certificate_info_title'] = _('About {company_name} Certificates').format(
+        company_name=company_name
+    )
+
+    cert_info_honor_code_html = "<a href='{tos_url}'>{company_name} Honor Code</a>".format(
+        tos_url=context.get('company_tos_url'),
+        company_name=company_name
+    )
+    cert_info_verified_html = "<a href='{verified_cert_url}'>verifying your identity</a>".format(
+        verified_cert_url=context.get('company_verified_certificate_url')
+    )
+    # Translators:
+    context['certificate_info_description'] = _("{company_name} acknowledges achievements through certificates, which "
+                                                "are awarded for various activities {company_name} students complete "
+                                                "under the {honor_code_html}.  Some certificates require completing "
+                                                "additional steps, such as {verifying_html}.").format(
+        company_name=company_name,
+        honor_code_html=cert_info_honor_code_html,
+        verifying_html=cert_info_verified_html
+    )
+
+    # Translators:
+    context['certificate_type_title'] = _('{certificate_type} Certfificate').format(
+        certificate_type=context.get('certificate_type')
+    )
+
+    # Translators:
+    context['certificate_verify_title'] = _("How {company_name} Validates Student Certificates").format(
+        company_name=company_name
+    )
+
+    # Translators:
+    context['certificate_verify_description'] = _("Certificates issued by {company_name} are signed by a gpg key so "
```

Comment:
Might want to link to an explanation about gpg keys, public keys, etc. It's totally cool to just have a resource in the translators' comment so they can read & understand.